### PR TITLE
Test auth

### PR DESCRIPTION
**Pull request by hyfi06**
add test of routes and services auth

Refs #104

### CODE REVIEW

**Comment by hyfi06** [comment]
Mental note: tests must run without .env

**Comment by ealexisaraujo** [comment]
I ran test:

Screen Shot 2020-05-27 at 15 18 10

File: test/auth.routes.test.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    '../../config': configMock,
+  });
+
+  const request = testServer(route);
+
+  this.beforeEach(() => {
+    this.authenticate = sinon.stub(passport, 'authenticate')
+      .callsFake((strategy, callback) => {
+        if (strategy === 'basic') {
+          return function (req, res, next) {
```
Comment (ealexisaraujo) [inline test/auth.routes.test.js:24]: You are not using res and next, I ran the test without res and next, why are you using this ? 
Screen Shot 2020-05-27 at 15 18 10

Comment (ealexisaraujo) [reply test/auth.routes.test.js:24]: @hyfi06 👆

**Pull request merged**